README.md

```diff
@@ -1,5 +1,28 @@
 # Wikidata Reference Island
 
+## Installation and Running Tests
+We can create a python virtual environment.
+This has the advantage of keeping dependencies from this project separate from your system.
+
+Create a venv in a folder `./venv`
+`python3 -m venv venv`
+
+Active the virtual environment:
+`source venv/bin/active`
+
+Install this library and test dependencies:
+`pip3 install .`
+`pip3 install -e .[tests]`
```

Comment:
So we don't need to run `setup.py` directly in this case, right?

Reply:
line 14 is basically the same.